web-test-runner.config.mjs

```diff
@@ -1,5 +1,6 @@
 import { importMapsPlugin } from '@web/dev-server-import-maps';
 import { defaultReporter } from '@web/test-runner';
+import { chromeLauncher } from '@web/test-runner-chrome';
```

Comment:
Did you want to commit this? Sneaky way of getting this in! 😁

Reply:
In light of #922 , I don't think we should add this.

Reply:
damn

Reply:
(sorry about that, just me doing yet another round of test and forgot to remove it again :/)